spec/acceptance/azure_setup_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper_acceptance'
+
+
+describe 'azure class' do
+  azurepackages = ['retries', 'azure', 'azure_mgmt_compute', 'azure_mgmt_storage','azure_mgmt_resources', 'azure_mgmt_network', 'hocon']
+
+#  case fact('osfamily')
```

Comment:
There are two sets of packages, the gem packages that every OS needs, and the dependencies to compile nokogiri, which vary based on OS.  I never got the acceptance tests running correctly, hence commenting out the case statement here.  Need to tweak it more but the hardcoded 2015.2 version seems like the nodesets are busted to begin with.